pkg/maps/ipcache/ipcache.go

```diff
@@ -41,7 +41,7 @@ const (
 	// (/8) will always be inserted, which is what the first parameter
 	// denotes. The CIDR for the CIDRSet is the second parameter, and the
 	// exception is the third parameter.
-	maxPrefixLengths = 2 - 8 + 32
+	maxPrefixLengths = 2 - 8 + 26
```

Comment:
This is actually now restricting us to 20 unique prefixes, while the `lxc_config.h` is stating that we should support ~26.